hello.cpp

```diff
@@ -1,28 +1,45 @@
+#include <cstring>
 #include <launchdarkly/api.hpp>
 
 const char *username = "jeff@example.com";
```

Comment:
I think we can remove this line, per comment below on having this be Sandy/example-user-key.